examples/demo_runner.py

```diff
@@ -314,6 +314,11 @@ def init_common(self):
 
         self._sim = habitat_sim.Simulator(self._cfg)
 
+        # test recompute navmesh
+        navmesh_settings = habitat_sim.NavMeshSettings()
```

Comment:
```suggestion
        navmesh_settings = habitat_sim.nav.NavMeshSettings()
```

to follow that it's exposed through the `nav` package?